docs/DEVELOPMENT.md

```diff
@@ -48,32 +48,49 @@ After making your changes, go in the deltachat/electron Dev-console and press `F
 
 ### Code Style <a id="code-style"></a>
 
-We use a combination of [`ESLint`](https://eslint.org) and [`Prettier`](https://prettier.io/) to cover linting/formatting of different types of files.
+#### Checking Code Style
```

Comment:
Putting back the previous check and format sections but with more fine grained explanation of what's being done during check and fix.